assets/js/components/ReportTable.stories.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * ReportTable Component Stories.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import ReportTable from './ReportTable';
+import Layout from './layout/Layout';
+import Link from './Link';
+import { CORE_MODULES } from '../googlesitekit/modules/datastore/constants';
+import {
+	createTestRegistry,
+	provideModules,
+	provideModuleRegistrations,
+	WithTestRegistry,
+} from '../../../tests/js/utils';
+
+const Template = ( args ) => <ReportTable { ...args } />;
+
+export const ReportTableBasic = Template.bind( {} );
+ReportTableBasic.storyName = 'Basic';
+const columns = [
+	{
+		title: 'Name',
+		description: 'Module name',
+		primary: true,
+		Component: ( { row } ) => (
+			<Link href={ row.homepage } children={ row.name } external />
+		),
+	},
+	{
+		title: 'Description',
+		description: 'Module description',
+		field: 'description',
+	},
+	{
+		title: 'Icon',
+		Component: ( { row } ) => row.Icon && <row.Icon width={ 33 } />,
+	},
+];
+const registry = createTestRegistry();
```

Comment:
@jimmymadon This logic, till L60 is specific to the `ReportTableBasic` story. So let's put the logic in the story's `decorators` like this:
```js
ReportTableBasic.decorators = [
	( Story ) => {
		const registry = createTestRegistry();
		provideModules( registry );
		provideModuleRegistrations( registry );
		const modules = registry.select( CORE_MODULES ).getModules();
		const rows = Object.values( modules );
		const columns = [
			{
				title: 'Name',
				description: 'Module name',
				primary: true,
				Component: ( { row } ) => (
					<Link
						href={ row.homepage }
						children={ row.name }
						external
					/>
				),
			},
			{
				title: 'Description',
				description: 'Module description',
				field: 'description',
			},
			{
				title: 'Icon',
				Component: ( { row } ) => row.Icon && <row.Icon width={ 33 } />,
			},
		];

		return <Story rows={ rows } columns={ columns } />;
	},
];
```